webroot/panel/account.php

```diff
@@ -3,20 +3,37 @@
 require_once "../../resources/autoload.php";
 
 use UnityWebPortal\lib\UnitySite;
+use phpseclib3\Crypt\PublicKeyLoader;
 
 require_once $LOC_HEADER;
 
+$invalid_ssh_dialogue = "<script type='text/javascript'>
+alert('Invalid SSH Key. Try again.');
+</script>";
+
 if ($_SERVER['REQUEST_METHOD'] == "POST") {
     switch ($_POST["form_type"]) {
         case "addKey":
             $added_keys = array();
 
             switch ($_POST["add_type"]) {
                 case "paste":
-                    array_push($added_keys, $_POST["key"]);
+                    try {
+                        PublicKeyLoader::load($_POST["key"], $password = false);
+                        array_push($added_keys, $_POST["key"]);
+                    } catch (Exception $e) {
+                        echo $invalid_ssh_dialogue;
+                    }
```

Comment:
I'd prefer if this block of code is written in a function on the top of this file, then used twice in the code up here. Something like `testValidSSHKey($key_str)` which returns true/false

Reply:
Added the function but can't fix one codesniffer warning